config/sgp4_scanner.json.empty

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2014-2016 Kartik Kumar (me@kartikkumar.com)
+// Copyright (c) 2014-2016 Abhishek Agrawal (abhishek.agrawal@protonmail.com)
+// Distributed under the MIT License.
+// See accompanying file LICENSE.md or copy at http://opensource.org/licenses/MIT
+
+// Configuration file for D2D "sgp4_Scanner" application mode.
+{
+    "mode"                      : "sgp4_scanner",
+
+    // Set path to TLE catalog file.
+    "catalog"                   : "../data/catalog/",
+
+    // Set Velocity cut-off in km/s.
```

Comment:
One more sentence below this explaining what the cut-off is for would be helpful, so users who see this config file know what the option does.